controllers/apply/awards-for-all/form.test.js

```diff
@@ -329,7 +329,7 @@ describe('Project details', () => {
                     endDate: { day: 1, month: 1, year: 2030 }
                 }
             },
-            [expect.stringMatching(/Date you end the project must be within/)]
+            [expect.stringMatching(/Date you start the project must be after/)]
```

Comment:
Not sure I follow why this has changed?

Reply:
Discussed this in-person but the gist of it is: this bug was causing ambiguous validation rules anyway, but also because the date 1st January 2020 is now an invalid start date (as of the last day or so), so this test returns a different result than it used to. Have updated it to make the dates dynamic.